backend/services/core/courses/courses.middleware.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const router = express.Router();
+const url = require('url');
+const moment = require('moment');
+
+module.exports = (server) => {
+
+	router.get('/courses', (req, res, next) => {
+		let url_parts = url.parse(req.originalUrl, true);
+		const query = url_parts.query;
+		const from = parseInt(query.start, 10) || 0;
+		let	to = from + parseInt(query.count, 10);
+		const sort = query.sort;
+		const id = query.id;
+		let	courses = server.db.getState().courses;
+
+		if (!!query.textFragment) {
+			courses = courses.filter((course) => course.name.concat(course.description).toUpperCase().indexOf(query.textFragment.toUpperCase()) >= 0);
+		}
+
+		if(sort) {
```

Comment:
Do you run prettier before code comiting?

Reply:
i havent work with backend at all.

Reply:
I am not talking about working with BE, I am talking about code formatting